pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -640,8 +642,8 @@ func (k *Bootstrapper) UpdateCluster(cfg config.KubernetesConfig) error {
 		}
 	}
 
-	if err := k.c.Run(`sudo systemctl daemon-reload && sudo systemctl start kubelet`); err != nil {
-		return errors.Wrap(err, "starting kubelet")
+	if rr, err := k.c.RunCmd(exec.Command("/bin/bash", "-c", "set -x;sudo systemctl daemon-reload && sudo systemctl start kubelet")); err != nil {
+		return errors.Wrapf(err, "starting kubelet, output: %s", rr.Output())
```

Comment:
stderr is likely more important for error messages. Perhaps include by default?

Reply:
do you mean in the RunResults.err , wrap the err with stderr ?